src/nuget/Microsoft.xunit.netcore.extensions.nuspec

```diff
@@ -14,30 +14,30 @@
     <description>This package provides things like various traits and discovers like OuterLoop/ActiveIssue that are used by .NET Core framework test projects.</description>
     <copyright>Copyright © Microsoft Corporation</copyright>
     <dependencies>
-      <group targetFramework="netstandard1.3">
-        <dependency id="System.Diagnostics.Debug" version="4.0.11-rc3-24117-00" />
-        <dependency id="System.IO" version="4.1.0-rc3-24117-00" />
-        <dependency id="System.Linq" version="4.1.0-rc3-24117-00" />
-        <dependency id="System.Reflection" version="4.1.0-rc3-24117-00" />
-        <dependency id="System.Reflection.Primitives" version="4.0.1-rc3-24117-00" />
-        <dependency id="System.Runtime" version="4.1.0-rc3-24117-00" />
-        <dependency id="System.Runtime.Extensions" version="4.1.0-rc3-24117-00" />
-        <dependency id="System.Runtime.Handles" version="4.0.1-rc3-24117-00" />
-        <dependency id="System.Runtime.InteropServices" version="4.1.0-rc3-24117-00" />
-        <dependency id="System.Runtime.InteropServices.RuntimeInformation" version="4.0.0-rc3-24117-00" />
-        <dependency id="System.Text.Encoding" version="4.0.11-rc3-24117-00" />
-        <dependency id="System.Threading" version="4.0.11-rc3-24117-00" />
-        <dependency id="System.Threading.Tasks" version="4.0.11-rc3-24117-00" />
+      <group targetFramework="netstandard1.0">
+        <dependency id="System.Diagnostics.Debug" version="4.0.11" />
+        <dependency id="System.IO" version="4.1.0" />
+        <dependency id="System.Linq" version="4.1.0" />
+        <dependency id="System.Reflection" version="4.1.0" />
+        <dependency id="System.Reflection.Primitives" version="4.0.1" />
+        <dependency id="System.Runtime" version="4.1.0" />
+        <dependency id="System.Runtime.Extensions" version="4.1.0" />
+        <dependency id="System.Runtime.Handles" version="4.0.1" />
+        <dependency id="System.Runtime.InteropServices" version="4.1.0" />
+        <dependency id="System.Runtime.InteropServices.RuntimeInformation" version="4.0.0" />
```

Comment:
Correct me if I'm wrong but I thought RuntimeInformation was only supported on netstandard1.1 and higher.